server/command_hooks.go

```diff
@@ -355,6 +363,12 @@ func getDialogWithoutElements(state string) model.Dialog {
 	}
 }
 
+func getDialogWithIntroductionText(introductionText string) model.Dialog {
+	dialog := getDialogWithSampleElements()
+	dialog.IntroductionText = introductionText
+	return dialog
```

Comment:
indentation here seems off?

Reply:
No, it's correct. That just a bug with GitHub.